src/mcp/types.py

```diff
@@ -646,6 +646,27 @@ class ImageContent(BaseModel):
     model_config = ConfigDict(extra="allow")
 
 
+class DataContent(BaseModel):
```

Comment:
Slightly confused didn't https://github.com/modelcontextprotocol/modelcontextprotocol/pull/356 get paused awaiting more evidence extra complexity is worth while.